src/main/java/frc/robot/subsystems/simulated/CANSparkMaxSimulated.java

```diff
@@ -10,13 +10,26 @@
  * the `get` method works properly.
  */
 public class CANSparkMaxSimulated extends CANSparkMax {
+    double m_dutyCycle = 0;
```

Comment:
This change probably isn't necessary, but I made the get / set logic in the simulated motor more explicit so if the CAN class changes, our behavior doesn't.